### PR TITLE
Updates EC2 Guide for 3.1 & Adds `aws` commands

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Adds the cross account instructions and also changes IAM images to commands. 

## Motivation and Context
Adds instruction on how to use cross-account roles.

## How Has This Been Tested?
`yarn server`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [ ] Fixing errata
- [x] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All tests have passed.

### CODE REVIEW

File: content/sensu-enterprise/3.1/guides/using-ec2.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+aws iam attach-role-policy --profile trusting --role-name cross-account-readonly --policy-arn "arn:aws:iam::111111111111:policy/cross-account-readonly"
+{{< /highlight >}}
+
+#### Trusted account
+
+1. Create 'cross-account-assumerole' IAM policy
+{{< highlight shell >}}
+aws iam create-policy --profile trusted --policy-name cross-account-assumerole --policy-document file://cross-account-assumerole-policy.json
+{{< /highlight >}}
+Note the ARN for this policy, needed for step 3
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.1/guides/using-ec2.md]: While I don't know if an official NOTE is needed here, it looks odd on the website.

**Comment by cwjohnston** [comment]
I’m not sure where in the guide this should be called out, but the `client.ec2.account` attribute needs to have a value which reflects which account should be used when the Sensu Enterprise server processes a keepalive event for the client. This is how the Enterprise server knows which role to assume for querying the EC2 API.

**Comment by asachs01** [comment]
AH RIGHT @cwjohnston

**Comment by asachs01** [comment]
@cwjohnston just added the config. Can you verify?

File: content/sensu-enterprise/3.1/guides/using-ec2.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+### Definitions
+
+Trusting account -- an AWS account where Sensu Clients are running. An
+IAM 'read-only' role is created here allowing ec2:DescribeInstances
+action, with a policy that allows the trusted account's instance role
+to assume the read-only role.
+
+For the purposes of this document:
+* The AWS account number for the trusting account is assumed to be `111111111111`
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.1/guides/using-ec2.md]: I think we need a new line above the bullet points

File: content/sensu-enterprise/3.1/guides/using-ec2.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+* The AWS CLI is assumed to be configured with a profile `trusting`
+
+Trusted account -- an AWS account where Sensu Enterprise is running. An
+EC2 instance is created here with an instance role attached allowing
+the instance to assume a role in the trusting account.
+
+For the purposes of this document:
+* The AWS account number for the trusted account is assumed to be `222222222222`
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.1/guides/using-ec2.md]: new line needed here as well

File: content/sensu-enterprise/3.1/guides/using-ec2.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+`region`: The `region` attribute is the region that the integration will use for queries about the instance's state. By default, this is set to `us-east-1` (the "Virginia" region).
+
+`allowed_instance_states`: The `allowed_instance_states` attribute determines the state that an instance can be in before it is automatically deregistered. By default, this is set to "running", but you can also set it to "running", "stopping", "stopped", "shutting-down", "terminated", "rebooting", and "pending". Any instance that is in a state not specified in the configuration will be deregistered.
+
+You can use the above example configuration and substitute your own access key and secret key in for the ones in the example. Let's continue and take a look at what is necessary at the client level for the EC2 integration to function properly.
+
+_NOTE: Don't forget to restart the Sensu Enterprise process via `systemctl restart sensu-enterprise` for the configuration changes to take effect._
+
+### Client Configuration
+
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.1/guides/using-ec2.md:65]: For cross-account usage, the client ec2 attributes need to include the account name, e.g. `{ "client": { "ec2": { "account": "sensu-testing" } } }`

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Comment by asachs01** [comment]
@cwjohnston check me again on the attributes?

File: content/sensu-enterprise/3.1/guides/using-ec2.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+      }
+    ]
+  }
+}{{< /highlight >}}
+
+Where the account in the ARN above is the account that will be queried. 
+
+#### Client Configuration
+
+Clients will also need to be configured to use a specific account. This is done by the `"name"` specified in `/etc/sensu/conf.d/ec2.json`. See the example below:
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.1/guides/using-ec2.md]: > This is done by the `"name"` specified in...

I think it should be the client.ec2.account attribute

How about this?

"This is done by the "account" attribute specified in `client.ec2` scope of `/etc/sensu/conf.d/ec2.json`"

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**